commcare_connect/organization/forms.py

```diff
@@ -30,8 +32,13 @@ class Meta:
         labels = {"user": "", "role": ""}
 
     def __init__(self, *args, **kwargs):
+        self.organization = kwargs.pop("organization")
         super().__init__(*args, **kwargs)
 
+        self.fields["user"].queryset = User.objects.exclude(
+            Q(email__isnull=True) | Q(memberships__organization=self.organization)
```

Comment:
nit: I think this would be slightly clearer as `User.objects.filter(email__isnull=False).exclude(memberships__organization=self.organization)` (combining `NOT` and `OR` like that can be a bit less easy to reason about that a simple `AND`)

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/232/commits/81d151fb86f62e7868d228e6661683b5ba0eb448